server/channels/store/sqlstore/post_store.go

```diff
@@ -2683,10 +2684,61 @@ func (s *SqlPostStore) GetParentsForExportAfter(limit int, afterId string, inclu
 			continue
 		}
 
+		flaggedMap, err := s.getFlaggedPostUsernames(rootIds)
```

Comment:
How much of a change would it be to get the flaggedBy users in the main query itself? It seems like anyways we are using a sub-query there. Doing another potentially heavy query again should be considered carefully.